traveller.yaml

```diff
@@ -1406,6 +1342,15 @@ definitions:
         items:
           $ref: '#/definitions/spatialConstraint'
 
+  productSetInformationUpdate:
+    description: Product Set Information Update
+    type: object
+    required:
+      - productSetId
```

Comment:
no productSetId property

Reply:
217d0ad2fdd36ad394b44744c6c88f0985f9d135 and db24fe8a181d8fd076106a59161a30fcc9cb7424